src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt

```diff
@@ -150,4 +152,31 @@ class PremisesController(
 
     return ResponseEntity.ok(lostBedsTransformer.transformJpaToApi(lostBed))
   }
+
+  override fun premisesPremisesIdCapacityGet(premisesId: UUID): ResponseEntity<List<DateCapacity>> {
+    val premises = premisesService.getPremises(premisesId)
```

Comment:
very very minor - I wonder if there is any point in letting the exception handler process these by throwing, since you are having to deal with ResponseEntity here anyway you could just do something like `return ResponseEntity.notFound(some error )`

Obviously you would have to restructure this sine you want to return early etc etc

Reply:
So we could do but because we're returning an https://datatracker.ietf.org/doc/html/rfc7807 problem+json response we'd end up with quite a lot of duplication everywhere constructing the response bodies